ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/checkpointsync/CheckpointSource.java

```diff
@@ -35,12 +33,10 @@ public class CheckpointSource implements Iterator<Hash> {
   private final AtomicBoolean isDownloading = new AtomicBoolean(false);
 
   public CheckpointSource(
-      final SyncState syncState,
-      final EthPeer ethPeer,
-      final BlockHeaderFunctions blockHeaderFunctions) {
+      final SyncState syncState, final BlockHeader blockHeader, final int checkpointWindowSize) {
```

Comment:
nit: would be nice if this arg name matched the field name

Reply:
@jframe do you think nbBlocks or checkpointWindowSize makes more sense here? i.e. rename the field or rename the arg name?